### ISSUE TITLE
Support for time.Time type

### ISSUE THREAD

**Issue by daisuke0925m** [Warashi/extfieldgen#10]
Can ExtFieldGen handle time.Time?

If `time.Time` is specified, it seems that model_time is generated as follows.

```go
extend type hoge @extraField(name: "CreatedAt", type: "time.Time", description: "CreatedAt")

```

```go
import model_time "github.com/path/to/model.time"


type hoge struct {
	// CreatedAt
	CreatedAt model_time.Time `json:"-"`
}
```

**Comment by Warashi** [comment]
Thank you for reporting this issue.
This plugin uses gqlgen's extra_fields feature.
ref; https://gqlgen.com/recipes/extra_fields/

So, can you try gqlgen's feature?
If the problem is reproduced with the native feature, please report the issue to gqlgen?

**Comment by Warashi** [comment]
@daisuke0925m 
I investigated this issue, and found bugs.
I fixed it in #11 , so please `go get -u` and try again.
